Ticket CC: visual overhaul

iiloni-umd opened the pull request:
#### What's this PR do?
Updates the Ticket CC management pages to conform to the design of the rest of helpdesk and make use of Bootstrap 4 components. See screenshots for before/after comparison. There are no functional changes, only visual.

#### Should this PR be reviewed?
Yes, I would like feedback, if any, on the new design. I would also like to know if there are any functional issues that should be addressed that I might not be aware about.

#### How should this be manually tested?
Visit the CC page of any ticket

#### What are the relevant tickets?
https://clearlyenergy-inc.monday.com/boards/7029256041/pulses/7318879928

#### Screenshots (if appropriate)
Main Page: 
| Before | After |
|--------|--------|
| ![image](https://github.com/user-attachments/assets/bf5830aa-80ae-4915-a18c-315d4889d2c3) | ![image](https://github.com/user-attachments/assets/d2e8ee8a-1068-4a4d-830a-6fba0c1f4020) |

Add Page:
| Before | After |
|--------|--------|
| ![image](https://github.com/user-attachments/assets/1487ac9a-b51a-43ac-afd8-91eda0202344) | ![image](https://github.com/user-attachments/assets/987ca089-094e-4d1c-855e-9d1111bb38dc)| 

Delete Page:
| Before | After |
|--------|--------|
| ![image](https://github.com/user-attachments/assets/5d6daf5c-7660-40cd-ac23-6749da9a7a76) | ![image](https://github.com/user-attachments/assets/9566e013-96af-478f-99ac-b6cef41bf941)|

ebeers-png commented:
It already looks so much better, thank you!! I have some functionality requests as well:
- On the add page, combine the non-user email and user tabs together into just one form; maybe make it a "Search for a user in BEAM (via dropdown) or type the email address to copy:" form.
- On the add page, view/update checkboxes should be toggled by default. (I honestly question whether anyone uses either of those options anyway - maybe we should just remove them from Helpdesk? Or at least from the frontend, and just set them to "true" by default?)
- On the main page, change "Delete" column header to say "Remove," and add a column for "Staff" that's checked when the user is_helpdesk_staff

iiloni-umd commented:
Sure thing! For the select user or enter email part, I made both fields visible and now selecting a user from the dropdown disables the email field - that way it's clear only 1 of those fields is accepted.

Updated main page: 
![image](https://github.com/user-attachments/assets/e69ce1ec-cfe0-47a1-ad22-d75e02d116d5)

Updated add page:
![image](https://github.com/user-attachments/assets/9a4a5882-47b5-4087-8345-6ead0c7051fc)
